examples/linear_model/plot_robust_fit.py

```diff
@@ -22,14 +22,20 @@
 - RANSAC is good for strong outliers in the y direction
 
 - TheilSen is good for small outliers, both in direction X and y, but has
-  a break point above which it performs worst than OLS.
+  a break point above which it performs worse than OLS.
+
+- HuberRegressor should not differ much in performance to both RANSAC
```

Comment:
this seems to be slightly in conflict in what the user guide says. also: is performance here accuracy or training time?